lib/mobile_app_backend/application.ex

```diff
@@ -29,6 +29,9 @@ defmodule MobileAppBackend.Application do
         {MBTAV3API.RepositoryCache, []},
         MBTAV3API.Supervisor,
         {MobileAppBackend.Health.FinchPool, pool_name: Finch.CustomPool},
+        # TODO: Enable this once DB is created in deployed environments
+        #  MobileAppBackend.Repo,
+        # {Ecto.Migrator, repos: Application.fetch_env!(:mobile_app_backend, :ecto_repos)},
```

Comment:
This is the key piece to actually connect to the DB once it has been created in dev & prod.